vignettes/data_dictionary.Rmd

```diff
@@ -123,17 +123,145 @@ DT::datatable(dd_companies_included)
 
 ## Aggregated PACTA metrics
 
-dd_company_technology_deviation_tms,
-  dd_company_alignment_net_tms,
-  dd_company_alignment_bo_po_tms,
-  dd_company_alignment_net_sda,
-  dd_company_exposure_bo_po_aggregate_alignment,
-  dd_company_exposure_net_aggregate_alignment,
-  dd_loanbook_exposure_bo_po_aggregate_alignment,
-  dd_loanbook_exposure_net_aggregate_alignment,
-  dd_data_sankey,
-  dd_data_scatter_alignment_exposure,
-  dd_data_scatter_sector,
-  dd_data_scatter_sector_animated,
-  dd_data_timeline_net,
-  dd_data_timeline_bo_po
+The aggregated PACTA metrics are also run across all input banking books. The calculations produce the net aggregate alignment metric, which is defined in `pacta.multi.loanbook.analysis` and allows producing the corresponding plots using `pacta.multi.loanbook.plot`. Results are grouped at the level defined by the `by_group` parameter.
+
+### Company technology deviation
+
+For each company in the analyzed banking books, shows the deviation of the technology build-out in the final year of the analysis from the corresponding allocated scenario value. This is an intermediate result that is further processed in the calculation of the net aggregate alignment metric. Only available for sectors, which have technology level calculations using the target market share, namely `automotive, coal, oil and gas, power`.
+
+```{r dd_company_technology_deviation_tms}
+dd_company_technology_deviation_tms <- dplyr::filter(data_dictionary, .data[["dataset"]] == "company_technology_deviation_tms")
+DT::datatable(dd_company_technology_deviation_tms)
```

Comment:
Is {DT} really necessary here? If not, probably an unnecessary complication that might add confusion. If so, possibly could use some explanation of why it's needed / what it does.

Reply:
ack, sorry... I was thinking of {data.table}... {DT} is being used to make fancy formatted tables in the HTML output.

Should we add {DT} to Suggests for this?

Reply:
ack! you already did 🤦🏻 never mind LOL